src/backend/web/app.js

```diff
@@ -27,12 +25,16 @@ app.use(helmet());
 app.use(cors());
 app.options('*', cors());
 
-// TODO: decide if we should do resave=false, https://www.npmjs.com/package/express-session#resave
+// Setup session and passport for authentication
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+authentication.init(passport);
 app.use(
   session({
     store: new RedisStore({ client: redis }),
-    secret,
+    secret: process.env.SECRET || `telescope-has-many-secrets-${Date.now()}!`,
```

Comment:
Shouldn't the variable on the right (telescope-has-many-secrets) have a naming convention that is a bit easier to understand? or is it fine because it's a secret.

Reply:
It's just a string we use to make the hashes for the cookies more random.  It doesn't matter what it is.  People often use stupid strings in here, just to make it so it's not guessable.  You want a long string, basically.